lib/improver/cli/estimate_emos_coefficients.py

```diff
@@ -29,186 +29,24 @@
-    parser.add_argument('--tolerance', metavar='TOLERANCE',
-                        type=np.float32, default=0.01,
-                        help='The tolerance for the Continuous Ranked '
-                             'Probability Score (CRPS) calculated by the '
-                             'minimisation. The CRPS is in the units of the '
-                             'variable being calibrated. The tolerance is '
-                             'therefore representative of how close to the '
-                             'actual value are we aiming to forecast for a '
-                             'particular variable. Once multiple iterations '
-                             'result in a CRPS equal to the same value '
-                             'within the specified tolerance, the '
-                             'minimisation will terminate.')
-    parser.add_argument('--landsea_mask', metavar="LANDSEA_MASK", default=None,
-                        help="The netCDF file containing a land-sea mask on "
-                             "the same domain as the historic forecast and "
-                             "truth data. Land points are specified by ones "
-                             "and sea points are specified by zeros.")
-    args = parser.parse_args(args=argv)
-
-    # Load Cubes
-    historic_forecast = load_cube(args.historic_filepath, allow_none=True)
-    truth = load_cube(args.truth_filepath, allow_none=True)
-    landsea_mask = load_cube(args.landsea_mask, allow_none=True)
-
-    combined = (load_cubelist(args.combined_filepath)
-                if args.combined_filepath else None)
-    historic_forecast_dict = (
-        load_json_or_none(args.historic_forecast_identifier))
-    truth_dict = load_json_or_none(args.truth_identifier)
-
-    # Process Cube
-    coefficients = process(historic_forecast, truth, combined,
-                           historic_forecast_dict, truth_dict,
-                           args.distribution, args.cycletime, landsea_mask,
-                           args.units, args.predictor_of_mean,
-                           args.tolerance, args.max_iterations)
-    # Save Cube
-    # Check whether a coefficients cube has been created. If the historic
-    # forecasts and truths provided did not match in validity time, then
-    # no coefficients would have been calculated.
-    if coefficients:
-        save_netcdf(coefficients, args.output_filepath)
-
-
-def process(historic_forecast, truth, combined, historic_forecast_dict,
-            truth_dict, distribution, cycletime, landsea_mask, units=None,
-            predictor_of_mean='mean', tolerance=1, max_iterations=1000):
-    """Module for estimate coefficients for Ensemble Model Output Statistics.
+@cli.clizefy
+@cli.with_output
+def process(*cubes: cli.inputcube,
+            distribution,
+            truth_attribute,
+            cycletime,
+            units=None,
+            predictor_of_mean='mean',
+            tolerance: float = 1,
```

Comment:
```suggestion
            tolerance: float = 0.01,
```

Reply:
but I'd probably leave it more explicit in the tests, just so that we can keep it synchronised with the more global `acc.LOOSE_TOLERANCE`

Reply:
@gavinevans, what you think about this?

Reply:
@tjtg opened an issue about tolerances, so I suggest we answer this question there and merge this into the feature branch to complete clize porting work.